charts/govuk-jobs/templates/content-store-mongo-to-postgres-cronjob.yaml

```diff
@@ -35,14 +35,18 @@ spec:
               emptyDir: {}
           initContainers:
             - name: export-mongo-data
-              image: "content-store:release"
+              image: "{{ .Values.images.DraftContentStore.repository }}:{{ .Values.images.DraftContentStore.tag }}"
```

Comment:
These values will be coming from [here](https://github.com/alphagov/govuk-helm-charts/blob/c55a034/charts/app-config/templates/govuk-application.yaml#L36) and won't be varying by draft/non-draft (since draft and non-draft are rolled out together as one Argo CD `Application`).

Similarly below, `DraftContentStorePostgresqlBranch` becomes `ContentStorePostgresqlBranch` because the template is taking the string from `imageValues` (from values-integration.yaml L1149 above, camelcasing it and filling in the image tag etc. from the correspondingly named file in ../charts/app-config/image-tags/...).
```suggestion
              image: "{{ .Values.images.ContentStore.repository }}:{{ .Values.images.ContentStore.tag }}"
```

(Apologies for the way this oddball `govuk-jobs` chart has ended up — it's become a bit of an ugly example to extend already.)